tools/fluent/brand_names.md

```diff
@@ -0,0 +1,16 @@
+# Brand names
+
+Brand names, like Firefox or Sync, are stored as *private messages* in dedicated files, and shared across applications.
+
+Unless indicated otherwise in comments, the [current policy](https://www.mozilla.org/styleguide/communications/translation/) is that brand names can’t be:
+* Declined to adapt to grammatical case.
```

Comment:
Did we tighten this additionally? I recall that pre- and suffixes were OK, as long as they didn't cut into the brand name. I.e., Firefoxa was OK, Firefoksa not.

Reply:
The policy sucks, but IMO it's pretty clear, we just never enforced it.
https://www.mozilla.org/en-US/styleguide/communications/translation/

> In some languages, depending on the grammatical case of the word, it may need to have a different ending or be otherwise rewritten to make sense. In these instances, please rewrite the sentence instead to keep the brand name unchanged.

Reply:
:(

Reply:
This policy does is wayyy easier to implement for web content than for UI (due to space constrains).
Is there any progress on communication with legal about changing that?

Reply:
It's on @peiying2 to start the discussion with both branding and legal, noting that the previous point of references in Legal just left (but I keep nagging her nonetheless ;-)).